Modules/booking/Model/BkCalendarEntry.php

```diff
@@ -904,6 +910,12 @@ public function journal(int $id_space, int $id_user, int $max=100) {
         return $res->fetchAll();
     }
 
+    public function blockedEntries($id_space) {
+        $sql = "SELECT * FROM bk_calendar_entry WHERE reason>0 AND id_space=? AND deleted=0 ORDER BY start_time DESC";
+        $req = $this->runRequest($sql, array($id_space));
+        return $req->fetchAll();
+    }
+
     public function computeDuration($id_space, $booking) {
```

Comment:
clicking on edit on a blocked resource line in bookingblock/{id_space} will throw the PfmException('no pricing found for client '.$id_client) line 935. We could use a $booking['reason'] != 0 condition to handle this case then access to the reservationedit page